packages/web/src/components/CheckboxField/_CheckboxField.scss

```diff
@@ -0,0 +1,81 @@
+@use 'theme';
+@use '../../tools/typography';
+@use '../../tools/accessibility';
+
+.CheckboxField {
+    display: inline-flex;
+    min-width: 0;
+}
+
+.CheckboxField__text {
+    margin-left: theme.$text-offset;
+}
+
+.CheckboxField__label {
+    @include typography.generate(theme.$label-typography);
+
+    margin-bottom: theme.$spacing;
+    color: theme.$label-color-default;
+}
+
+.CheckboxField__label--hidden {
+    @include accessibility.hide-text();
+}
+
+.CheckboxField__label--required::after {
+    content: '*';
+    color: theme.$label-required-color;
+}
+
+.CheckboxField__input {
+    appearance: none;
+    width: theme.$input-width;
+    height: theme.$input-width;
+    margin: 3px;
+    color: theme.$input-unselected-color;
+    background-position: center;
+    background-size: contain;
+    background-repeat: no-repeat;
+    border: 2px solid currentColor;
+    border-radius: 2px;
+    flex-shrink: 0;
+
+    &:checked {
+        color: theme.$input-selected-color;
+        background-image: theme.$input-checked-mark;
+        background-color: theme.$input-selected-color;
+    }
+
+    &:indeterminate {
+        color: theme.$input-selected-color;
+        background-image: theme.$input-indeterminate-mark;
+        background-color: theme.$input-selected-color;
+    }
+}
+
+.CheckboxField__message {
+    @include typography.generate(theme.$message-typography);
+
+    margin-top: theme.$spacing;
+    color: theme.$message-color-default;
+}
+
+.CheckboxField--error .CheckboxField__message {
```

Comment:
Can you please add ` > ` everywhere? We know the markup and this improves specificity.

Reply:
There is an additional class between them, so `.CheckboxField--disabled > .CheckboxField__text > .CheckboxField__label` is ok?

Reply:
Yes, thanks!